src/Symfony/Component/Form/Util/ServerParams.php

```diff
@@ -43,7 +43,7 @@ public function hasPostMaxSizeBeenExceeded()
      *
      * @return int|null The maximum post size in bytes
      */
-    public function getPostMaxSize()
+    public function getPostMaxSize(): ?int
```

Comment:
this seems to fail and return float on appveyor

Reply:
thanks, updated